Basic/DyanmicPrefabs/Assets/Scripts/05_API Playground (All Post-Connection Time Use Cases)/APIPlayground.cs

```diff
@@ -0,0 +1,374 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using Random = UnityEngine.Random;
+
+namespace Game.APIPlayground
+{
+    /// <summary>
+    /// This class serves as the playground of the dynamic prefab loading use cases. It integrates API from this sample
+    /// to use at post-connection time such as: connection approval for syncing late-joining clients, dynamically
+    /// loading a collection of network prefabs on the host and all connected clients, synchronously spawning a
+    /// dynamically loaded network prefab across connected clients, and spawning a dynamically loaded network prefab as
+    /// network-invisible for all clients until they load the prefab locally (in which case it becomes visible to the
+    /// client).
+    /// <remarks>
+    /// Assumption: Addressables are loadable, ie when the client tries to load it - it will not fail.
+    /// Improvement ideas:
+    /// - it's possible to have more advanced logic that would for instance kick players that are consistently failing
+    /// to load an addressable
+    /// - Addressable guid list could be compressed before being sent
+    /// - Instead of Addressable guids the peers could exchange a `short` index that would refer to Addressables in some
+    /// kind of a list stored in a ScriptableObject. That would reduce the amount of data that's being exchanged quite
+    /// drastically.
+    /// Last note: this sample does not cover the case of addressable usage when the client is loading custom visual
```

Comment:
I'd write this in the readme. And give some more details on this, the few lines here are unclear.

Reply:
This has been moved to this [PR](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/pull/93) instead.

Reply:
where? I can't find that text